src/visualizers/VisualizerDifferences.ts

```diff
@@ -37,13 +41,20 @@ class VizDifferences extends VisualizerDefault implements VisualizerInterface {
 		return new ValidationStatus(true);
 	}
 
-	drawDifferences(n: bigint, lvls: number, sequence: SequenceInterface) {
+	initialize(sketch: p5, seq: SequenceInterface): void {
+		super.initialize(sketch, seq);
+		if (seq.last - seq.first < this.settings.levels)
+		{
+			throw Error(`Sequence ${seq.name} has too few entries`
```

Comment:
There won't be a space between "entries" and "levels". That is, you'll get something like:

```
Sequence foo has too few entriesfor bar levels.
```

Reply:
My mistake, great catch. Will fix.